modules/nw-egress-router-about.adoc

```diff
@@ -0,0 +1,117 @@
+//
+// * networking/egress_router/using-an-egress-router.adoc
+
+[id="nw-egress-router-about_{context}"]
+= About an egress router
+
+The {product-title} egress router runs a service that redirects traffic to a
+specified remote server, using a private source IP address that is not used for
+anything else. The service allows pods to talk to servers that are set up
+to only allow access from allowed IP addresses.
+
+[NOTE]
+====
+The egress router is not intended for every outgoing connection. Creating large
+numbers of egress routers can push the limits of your network hardware. For
+example, creating an egress router for every project or application could exceed
+the number of local MAC addresses that the network interface can handle before
+falling back to filtering MAC addresses in software.
+====
+
+[IMPORTANT]
+====
+The egress router is not compatible with Amazon AWS, Azure Cloud, or any other cloud platform that does not support layer 2 manipulations due to their incompatibility with macvlan traffic.
+====
+
+[id="nw-egress-router-about-modes_{context}"]
+== Egress router modes
+
+In _redirect mode_, the egress router sets up iptables rules to redirect traffic
+from its own IP address to one or more destination IP addresses. Client pods
+that want to make use of the reserved source IP address must be modified to
+connect to the egress router rather than connecting directly to the destination
+IP.
+
+In _HTTP proxy mode_, the egress router runs as an HTTP proxy on port `8080`.
+This only works for clients talking to HTTP or HTTPS-based services, but usually
+requires fewer changes to the client pods to get them to work. Programs can be
+told to use an HTTP proxy by setting an environment variable.
+
+In _DNS proxy mode_, the egress router runs as a DNS proxy for TCP-based
+services from its own IP address to one or more destination IP addresses. Client
+pods that want to make use of the reserved, source IP address must be modified
+to connect to the egress router rather than connecting directly to the
+destination IP. This ensures that external destinations treat traffic as though
+it were coming from a known source.
+
+Redirect mode works for all services except for HTTP and HTTPS. For HTTP and
+HTTPS services, use HTTP proxy mode. For TCP-based services with IP addresses
+or domain names, use DNS proxy mode.
+
+[id="nw-egress-router-about-deployments_{context}"]
+== Deployment considerations
+
+The egress router adds a second IP address and MAC address to the primary
+network interface of the node.
+As a result, you might need to configure your hypervisor or cloud provider to allow the additional address.
+
+Red Hat OpenStack Platform::
+
+If you are deploying {product-title} on Red Hat OpenStack Platform, you must
+whitelist the IP and MAC addresses on your OpenStack environment, otherwise
+link:https://access.redhat.com/solutions/2803331[communication will fail]:
++
+----
+neutron port-update $neutron_port_uuid \
```

Comment:
This would be also available using the openstack client, which I'd use just for consistency. Overall, using specific project clients is considered legacy. You can check how this would be in https://docs.openstack.org/python-openstackclient/pike/cli/command-objects/port.html

Reply:
@danielmellado, what's a neutron port UUID? Does it belong in the `openstack port set` command? Is it `<port>` or is it what I'd expect a port to be, like `80` or `31337`?